src/main/java/sg/ncl/MainController.java

```diff
@@ -2283,11 +2303,21 @@ private void setSessionVariables(HttpSession session, String loginEmail, String
         session.setAttribute("sessionLoggedEmail", loginEmail);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "sessionLoggedEmail" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)